package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "cordova-plugin-android-fingerprint-auth":"https://github.com/OutSystems/cordova-plugin-fingerprint-aio.git#v3.0.1",
-    "cordova-plugin-touchid":"https://github.com/OutSystems/cordova-plugin-touchid.git#0.4.0-OS"
+    "cordova-plugin-touchid":"https://github.com/OutSystems/cordova-plugin-touchid.git#0.4.0-OS2"
```

Comment:
In line with what was communicated on the slack channel, these two lines should be removed completely as they are not necessary at all !

Reply:
ok, I will remove.